src/urbit/mar/chess/game/active.hoon

```diff
@@ -0,0 +1,34 @@
+/-  chess
+/+  chess
+=,  format
+|_  game=chess-game:chess
+++  grab
+  |%
+  ++  noun  chess-game:chess
+  --
+++  grow
+  |%
+  ++  noun  game
+  ++  json
+    %-  pairs:enjs
+    :~  ['gameID' [%s (scot %da game-id.game)]]
+        ['event' [%s event.game]]
+        ['site' [%s site.game]]
+        ['round' [%s (round-string:chess round.game)]]
+        ['white' [%s (player-string:chess white.game)]]
+        ['black' [%s (player-string:chess black.game)]]
+        ['result' [%s ?~(result.game '' u.result.game)]]
+        ::  default values. updated thru [%game @ta %updates ~] sub.
+        ['moves' [%a ~]]
+        ['archived' [%b %.n]]
```

Comment:
nit: I'd move this to the above section, as I don't consider this to be a default value